.ruby-version

```diff
@@ -1 +1 @@
-2.7.8
+3.2.2
```

Comment:
Not strictly necessary but I was unable to bundle with the Ruby version set ...

https://github.com/zendesk/stronger_parameters/blob/589cc2e35fee7d6b982d9366514e061126ac149b/.ruby-version#L1

This is the error I was getting ...

```
Bundler could not find compatible versions for gem "minitest":
  In snapshot (rails7.1.gemfile.lock):
    minitest (= 5.20.0)

  In rails7.1.gemfile:
    maxitest was resolved to 0.0.1, which depends on
      minitest (~> 5.4.2)

    minitest-rails was resolved to 7.1.0, which depends on
      minitest (~> 5.20)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

Problem goes away when I used a 3.2.2. Figured since we're testing for it then it's ok? 🤷🏽‍♂️ 

https://github.com/zendesk/stronger_parameters/blob/589cc2e35fee7d6b982d9366514e061126ac149b/.github/workflows/actions.yml#L13

Reply:
might run into trouble with `bundle_all` task ... but let's see